std_msgs/Header header
float64 scan_min 
float64 scan_max
